### DISCUSSION TITLE
Unable to click on a button using ScrollTo/ScrollDown

### DISCUSSION THREAD

**Discussion by coffeebanjo** [getgauge/taiko#2104] [Development]
I am trying to click on the 'Edit' button which is not visible on the mini browser window, I need to scrollTo/scrollDown the page to select the element. Using the below step, it is always clicking on the first available Edit button on the page instead of selecting the specified button.

Step:
await scrollTo(below(text("123456abc")), below(text('Line #')), toRightOf("4"), click('Edit'))

I am not sure if ScrollTo/ScrollDown works this way, but could you please help here? Thank you.

**Comment by coffeebanjo** [comment]
@NivedhaSenthil @zabil Any help here will be much appreciated. Thank you.

**Reply by zabil** [reply]
Can you share a sample that will make it easier to replicate this issue?
